server/rooms.ts

```diff
@@ -1238,6 +1241,7 @@ export class BasicChatRoom extends BasicRoom {
 			connection,
 			'|init|chat\n|title|' + this.title + '\n' + userList + '\n' + this.log.getScrollback() + this.getIntroMessage(user)
 		);
+		if (this.announcement) this.announcement.onConnect(user, connection);
 		if (this.poll) this.poll.onConnect(user, connection);
 		if (this.game && this.game.onConnect) this.game.onConnect(user, connection);
```

Comment:
I don't feel combining all three into `game` would be a good idea
How about just combining `pol`l and `announcement` instead?

Reply:
> I wonder how much people would mind needing to end polls to start a tour or something.
Well, I guess they would mind. 

And since polls and announcements can't exist at the same time, might as well merge em if you wanna avoid too much repetition